pagopa/pn-frontend
# feat/PN-11402 - Manage Empty State for each sender dashboard's component

## Short description
This code refactors the EmptyStatistics component to accept a custom message to be shown and uses this component inside the other statistics components to manage the empty state (no data to be shown after applying the start-end dates filter)

## List of changes proposed in this pull request
- refactor the EmptyStatistics component to make it reusable by other statistics components
- manage the empty state inside each statistics component

## How to test
Prerequisite: a IS_STATISTICS_ENABLE env var should be set to true in conf/config.json
- Navigate to the Statistics section and apply a narrow dates filter to obtain an empty data response for some of the statistics components. Verify the EmptyStatistics component is shown inside the specific component which has no data to be displayed.
- Login to manage the PA "Comune di Sappada" and verify the EmptyStatistics is applied for the whole statistics section.
